### PR TITLE
Bug/1523/labels do not take delta into account

### PR DESCRIPTION
**Pull request by RomanenkoVladimir**
# Labels do no take delta into account for height calculation

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_

closes: #1523 

## Description

- Current label implementation did not take label delta height into account when in delta mode, this fixes the issue
- Fix typo in variable name

### CODE REVIEW

**Review by JaiJoPa — APPROVED** [review]
LGTM

File: visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts [hunk 1]
```diff
@@ -89,14 +91,19 @@ export class CodeMapRenderService {
...
 		}
 	}
 
+	private getHighestNodeDelta(sortedNodes: Node[]) {
+		const sortedNodeValues = sortedNodes.map(node => node.height + Math.abs(node.heightDelta))
+		return Math.max(...sortedNodeValues)
+	}
+
 	private getHighestNode(sortedNodes: Node[]) {
 		const sortedNodeValues = sortedNodes.map(node => node.height)
```
Comment (BridgeAR) [inline visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts:108]:
```suggestion
	private getHighestNode(sortedNodes: Node[]) {
		const sortedNodeValues = sortedNodes.map(({ height, heightDelta }) => height + Math.abs(heightDelta ?? 0))
```

AFAIK the delta is only applied while in delta mode. Thus, it's simpler to just always add the value.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed 🐞
 
+-   Labels do not take delta height into account([#1523](https://github.com/MaibornWolff/codecharta/issues/1523))
```
Comment (BridgeAR) [inline CHANGELOG.md:18]:
```suggestion
-   Labels do not take delta height into account ([#1523](https://github.com/MaibornWolff/codecharta/issues/1523))
```

File: visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts [hunk 3]
```diff
@@ -38,15 +38,16 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 	}
 
 	//labels need to be scaled according to map or it will clip + looks bad
-	addLabel(node: Node, options: { showNodeName: boolean; showNodeMetric: boolean }, hightestNode: number) {
+	addLabel(node: Node, options: { showNodeName: boolean; showNodeMetric: boolean }, highestNode: number) {
```
Comment (BridgeAR) [inline visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts:41]: ❤️

**Review by BridgeAR — APPROVED** [review]
LGTM % comments

**Pull request merged**